libs/framework/include/celix/dm/Component.h

```diff
@@ -17,24 +17,21 @@
  * under the License.
  */
 
-#ifndef CELIX_DM_COMPONENT_H
-#define CELIX_DM_COMPONENT_H
+#pragma once
 
 #include "celix/dm/types.h"
 #include "dm_component.h"
 
 #include <map>
 #include <string>
 #include <vector>
+#include <atomic>
```

Comment:
I don't see this header being used anywhere, can this be removed or am I missing something?